Xamarin.Forms.Core/IElementConfiguration.cs

```diff
@@ -2,7 +2,7 @@
 namespace Xamarin.Forms
 {
 	// Don't make this generic covariant as it causes UWP performance to tank
```

Comment:
Leaving this comment here will make future devs' heads explode.

Reply:
haha oh yea
Once UI Tests pass I'll remove before merging